src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_pix2pix_zero.py

```diff
@@ -0,0 +1,843 @@
+# Copyright 2023 Pix2Pix Zero Authors and The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import inspect
+from typing import Any, Callable, Dict, List, Optional, Union
+
+import PIL
+import torch
+from transformers import (
+    AutoProcessor,
+    BlipForConditionalGeneration,
```

Comment:
when was BlipForConditionalGenreation added to `transformers`, which version?

Reply:
Ok only since 4.26.0 - we need to add some special code then when importing the pipeline

Reply:
4.26.1.

Reply:
Did you mean adding something like:

```py
import transformers

current_version = transformers.__version__

if current_version < "4.26.1":
   raise ValueError("This pipeline needs Transformers at least 4.26.1")
````